controllers/authorization.controller.js

```diff
@@ -57,8 +46,18 @@ const registerUser = async (req, res) => {
             roleId,
             userId: newUserId,
         });
+        const uuidForNewUser = await UserConfirmCodes.create({
+            userId: newUserId,
+        });
+        const link = `http://localhost:5000/api/confirm/${uuidForNewUser.dataValues.verifyCode}`;
```

Comment:
http://localhost:5000/ - in link is incorrect
when app will be deploed domain will change, move this variable to the env